src/fiatExchanges/FiatExchangeCurrencyBottomSheet.tsx

```diff
@@ -26,13 +27,15 @@ function FiatExchangeCurrencyBottomSheet({ route }: Props) {
   const cashInTokens = useCashInTokens()
   const cashOutTokens = useCashOutTokens(true)
   const spendTokens = useSpendTokens()
-  const tokenList =
+  const unsortedTokenList =
     flow === FiatExchangeFlow.CashIn
       ? cashInTokens
       : flow === FiatExchangeFlow.CashOut
       ? cashOutTokens
       : spendTokens
 
+  const tokenList = unsortedTokenList.sort(sortFirstStableThenCeloThenOthersByUsdBalance)
```

Comment:
can we wrap this in a useMemo? Also this needs unit tests

Reply:
would the was to unit test this be to do a snapshot comparison for what the component renders? I'm not quite sure how else to unit test this.

Reply:
you could select all token balance items (I think they should have a common testID) and assert the ordering with mocked input.